public/app.tsx

```diff
@@ -50,7 +50,10 @@ export const AiFlowDashboardsApp = (props: Props) => {
   return (
     <EuiPageTemplate
```

Comment:
nice, didn't know this existed vs <EuiPage> with more inner components

Reply:
A lot has changed in OpenSearch & OUI since the inception of the AD plugin :)